source/edge-server/configure.txt

```diff
@@ -551,34 +593,111 @@ look similar to:
 
 .. code-block:: shell
 
-   edgectl --query="store_id == 42"
+   edgectl config --global-query="store_id == 42"
+
+Sync a Subset of Documents from One Collection
+``````````````````````````````````````````````
+
+If the data source linked to your Edge Server-compatible service contains more
+than one collection, and you want to define a query for a specific collection,
+you can provide a mapping of the schema title for the collection, and a 
+relevant query. In this example, we set a query for the ``Inventory`` 
```

Comment:
```suggestion
relevant query. 

In this example, we set a query for the ``Inventory`` 
```

I would make this into a new paragraph just as a visual aid. Once I reached the part in the example about the store id, I forgot it was an example rather than universally true for Edge Server.